strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/search/MavenIndexerSearchProviderTest.java

```diff
@@ -148,10 +153,12 @@ public void shouldBeCapableToSearchByFullSha1Checksum(TestInfo testInfo)
     public void shouldBeCapableToSearchByClassNameByDefault(TestInfo testInfo)
             throws Exception
     {
-        artifactManagementService.validateAndStore(STORAGE0,
-                                                   getRepositoryName(REPOSITORY_RELEASES_WITH_CLASSNAMES_INDEXED,
-                                                                     testInfo),
-                                                   "org/carlspring/properties-injector/1.7/properties-injector-1.7.jar",
+    	
+    	RepositoryPath repositoryPath = repositoryPathResolver.resolve(STORAGE0,
```

Comment:
Cosmetic: Could you please follow the project's coding convention and align these method parameters properly (here and across thePR)?

Reply:
Done!